src/components/EditorHeader/ControlPanel.jsx

```diff
@@ -1835,30 +1913,42 @@ export default function ControlPanel({
                           if (menu[category][item].children) {
                             return (
                               <Dropdown
-                                style={{ width: "150px" }}
+                                style={{
+                                  width:
```

Comment:
```suggestion
                                className="min-w-36 max-w-64"
```

lets not do conditional styles